apps/foundation-demo/src/app/product/components/product-grid/product-grid.component.html

```diff
@@ -1,3 +1 @@
-<div class="product-grid">
-    <product-card *ngFor="let product of products" [product]="product"></product-card>
-</div>
\ No newline at end of file
+<product-card *ngFor="let product of products" [product]="product"></product-card>
```

Comment:
If I remove `host: { 'class' : 'product-grid' }` and `encapsulation: ViewEncapsulation.None`, the html will show an empty `<product-grid` host container and then the actual `<div class="product-grid`. Is there another way to prevent this from happening without adding the class to the host + using viewEncapsulation? It doesn't make sense to me to have the empty host container and then have to attach the class in the html as well.

Reply:
Just gonna chime in here with a cool little note about this:

```
:host {
 //grid-styles
}
```

will allow you to target the host element without breaching view-encapsulation. The only draw-back is that you don't have a "class" anywhere called `.product-grid`

Reply:
+1 on Damien's suggestion. But if you couldn't do that, I'd still say that it would be best to preserve view encapsulation even though you'd have the extra `<product-grid>` with nothing on it. It's the choice between having an extra layer in the html and defining somewhat global css styling. Luckily, Damien's solution gives us the best of both. In fact, I think we should change all components to use `:host`. It would remove a lot of lines of code. For some other PR though